package/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
We might want to use https://www.npmjs.com/package/@tsconfig/react-native as a config foundation

Reply:
I don't think it's really needed - comparing our config to what comes with @tsconfig/react-native as recommended I can see that the only difference we have is the `target` being set to `ESNext` vs `ES6` in the proposed package. If we can avoid adding yet another dependency let's do so.